task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +92,8 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a+b > c && a+c>b && c+b>a) return true;
+        else return false;
```

Comment:
You can return it in 1 row, like:
return a+b > c && a+c>b && c+b>a;
Expression will be truthfully or falsy :)